code/modules/clothing/factions/konyang.dm

```diff
@@ -9,8 +9,67 @@
 
 /obj/item/clothing/under/konyang/pink
 	name = "pink hanbok"
-	desc = "A traditional Konyang hanbok. This one is pink and white."
+	desc = "A traditional Konyanger hanbok. This one is pink and white."
 	icon_state = "hanbok_pink"
 	worn_state = "hanbok_pink"
 	item_state = "hanbok_pink"
 	slot_flags = SLOT_OCLOTHING | SLOT_ICLOTHING
+
+/obj/item/clothing/under/konyang/male
+	desc = "A traditional Konyanger hanbok tailored for men."
+	icon_state = "hanbok_male"
+	worn_state = "hanbok_male"
+	item_state = "hanbok_male"
+	slot_flags = SLOT_OCLOTHING | SLOT_ICLOTHING
```

Comment:
It already inherits this from the `/obj/item/clothing/under/konyang`. Not a requirement, just a note. Useful in case the original changes I suppose.